provelo_custom/views/hr_holidays_view.xml

```diff
@@ -27,19 +27,18 @@
         </field>
     </record>
 
-<!--    <menuitem fixme-->
-<!--        parent="hr_holidays.menu_hr_holidays_my_leaves"-->
-<!--        id="hr_holidays.menu_open_allocation_holidays"-->
-<!--        action="hr_holidays.open_allocation_holidays"-->
-<!--        groups="base.group_hr_manager"-->
-<!--        sequence="40"-->
-<!--    />-->
+    <!-- make this menuitem visible to all users -->
+    <record model="ir.ui.menu" id="hr_holidays.menu_hr_holidays_dashboard">
+        <field name="groups_id" eval="[(5,)]" />
+    </record>
+
+    <!-- make this menuitem visible only to hr managers -->
+    <record model="ir.ui.menu" id="hr_holidays.menu_open_allocation">
+        <field name="groups_id" eval="[(4, ref('hr.group_hr_manager'))]" />
```

Comment:
Did you consider using `(6, 0, [ref('hr.group_hr_manager')])` if you want to make sure hr_managers are the only ones that can access it ?

https://www.odoo.com/documentation/12.0/developer/reference/orm.html

Reply:
by default, the group is empty, meaning that anyone can access it. by adding at least one entry, it becomes a whitelist. the goal here is to have the menu item hidden for regular users, so unless that group is explicitly added, it will work.